tests/TestRunner/app/Marshalling/ObjCTypesTests.js

```diff
@@ -204,6 +204,14 @@ describe(module.id, function () {
         expect(interop.handleof(buffer)).toBe(data.bytes);
     })
 
+    it("MethodWithNSDecimalNumber", function () {
+        expect(NSDecimalNumber.maximumDecimalNumber() instanceof NSDecimalNumber).toBe(true);
```

Comment:
You've got me there, but wouldn't the error message would be more specific with the `toEqual` matcher if the expectation failed?